anomalib/utils/callbacks/__init__.py

```diff
@@ -74,6 +74,7 @@ def get_callbacks(config: Union[ListConfig, DictConfig]) -> List[Callback]:
         config.metrics.threshold.pixel_default if "pixel_default" in config.metrics.threshold.keys() else None
     )
     metrics_callback = MetricsConfigurationCallback(
+        config.dataset.task,
```

Comment:
I guess the same initialization is to be done when configuring `MetricsConfigurationCallback` in the new CLI

Reply:
done